abb_irb6600_support/urdf/irb6640_macro.xacro

```diff
@@ -1,212 +1,212 @@
 <?xml version="1.0"?>
 <robot xmlns:xacro="http://ros.org/wiki/xacro">
-<xacro:macro name="abb_irb6640" params="prefix">
+  <xacro:macro name="abb_irb6640" params="prefix">
     <!-- link list -->
     <link name="${prefix}base_link">
-        <collision name="collision">
-            <geometry>
-                <mesh filename="package://abb_irb6600_support/meshes/irb6640/collision/base_link.stl"/>
-            </geometry>
-            <material name="yellow">
-                <color rgba="0 1 1 1"/>
-            </material>
-        </collision>
-        <visual name="visual">
-            <geometry>
-                <mesh filename="package://abb_irb6600_support/meshes/irb6640/visual/base_link.stl"/>
-            </geometry>
-            <material name="orange">
-                <color rgba="1 0.43 0 1"/>
-            </material>
-        </visual>
+      <collision name="collision">
+        <geometry>
+          <mesh filename="package://abb_irb6600_support/meshes/irb6640/collision/base_link.stl"/>
+        </geometry>
+        <material name="yellow">
+          <color rgba="0 1 1 1"/>
+        </material>
```

Comment:
It might be nice to set `material` on `visual` instead of `collision` elements: afaik collision geometry will be assigned the `material` from the `visual` element, and right now all collision geometry is red (at least on my machine).

This is minor though.